craft_parts/executor/filesets.py

```diff
@@ -333,7 +334,7 @@ def _get_resolved_relative_path(relative_path: str, base_directory: str) -> str:
     return os.path.relpath(filename_abspath, base_directory)  # ty: ignore[invalid-return-type]
 
 
-def _normalize_entry(entry: str, default_partition: str) -> str:
+def normalize_entry(entry: str, default_partition: str) -> str:
```

Comment:
Should we include a note in the docstring that this isn't intended for use outside of craft-parts?